### PR TITLE
grc: forward port fixes from master XML to block.yaml definitions

### PR DESCRIPTION
**Pull request by noc0lour**
Some fixes to XML files happened on the master branch since next was created. Appropriate fixes are ported to YAML in this PR.
This will ease the merge from next back to master.

### CODE REVIEW

File: gr-digital/grc/digital_ofdm_sync_sc_cfb.block.yml [hunk 1]
```diff
@@ -15,6 +15,10 @@ parameters:
     options: ['False', 'True']
     option_labels: [Odd, Even]
     hide: part
+-   id: threshold
+    label: Threshold
+    dtype: real
+    default: '0.9'
```
Comment (haakov) [inline gr-digital/grc/digital_ofdm_sync_sc_cfb.block.yml]: Missing two asserts here: `fft_len > 0` and `cp_len >= 0`

From these lines: https://github.com/gnuradio/gnuradio/commit/ea468b8d1ff4283c8659487bcf4689e98d00d79e#diff-beb772f4d9d0b00937f98b136e97aef5R39

**Comment by noc0lour** [comment]
@marcusmueller don't merge until I squashed the fixup commit.

File: gr-digital/grc/digital_ofdm_sync_sc_cfb.block.yml (hunk 1 above)
Comment (noc0lour) [reply gr-digital/grc/digital_ofdm_sync_sc_cfb.block.yml]: Thanks :)

**Comment by haakov** [comment]
Also found some others, can be cherry-picked from [here](https://github.com/haakov/gnuradio/commit/d3598ab1ed6e3e72a8233d8c10b573ab42182f02).

(**_However_** I haven't had time to build this branch yet, so they are untested)

From the following commits:
https://github.com/gnuradio/gnuradio/commit/85e57a33e2f933927288cb61672a9037d5aff349
https://github.com/gnuradio/gnuradio/commit/28c1644e663b744417d3712f588dcadb98700f40
https://github.com/gnuradio/gnuradio/commit/1f8e58ea35db60bfd6c73cb71b9ea17552c276ef
https://github.com/gnuradio/gnuradio/commit/61c0382eb2935a6da6c81bd93c8b9c8ef2893df2#diff-168ab97bf4fea6a2593485f6b4dd9d41R36
https://github.com/gnuradio/gnuradio/commit/1d6051e20736d6e0254af91e0cb2ed07d7cc300b
https://github.com/gnuradio/gnuradio/commit/1d6051e20736d6e0254af91e0cb2ed07d7cc300b
https://github.com/gnuradio/gnuradio/commit/573d4c89f9ef4151b76eec8487c71adf4bbeebb0

**Comment by noc0lour** [comment]
Strange, they should have popped up during the merge. I take a look at them, thanks a lot!

**Comment by noc0lour** [comment]
@haakov I incorporated the missing xml bug fixes into the yml files you pointed out.

**Comment by marcusmueller** [comment]
closed by merging superset #1915

**Pull request closed without merging**